contracts/Warehouse.ligo

```diff
@@ -84,7 +84,7 @@ function assign (const params: assign_parameter; var storage: storage): return i
                     failwith ("NO_AVAILABLE_ITEM");
                 } else {
                     const inventory : contract (inventory_parameter) =
-                        case (Tezos.get_entrypoint_opt ("%assign_item", params.0) : option (contract (inventory_parameter))) of
+                        case (Tezos.get_entrypoint_opt ("%assign_item_inventory", params.0) : option (contract (inventory_parameter))) of
```

Comment:
could we keep the "assign_item" name for consistency with the other entry points?